src/data-management/Utils/get-RDF.ts

```diff
@@ -0,0 +1,5 @@
+import { readFileSync } from "fs";
+
+export function getRDFFromPath(path: string): string {
```

Comment:
```suggestion
export function getRDF(path: string): string {
```
Husk det gir sig selv ud fra parameteren